game/graphics/pipelines/opengl.cpp

```diff
@@ -186,6 +245,9 @@ static std::shared_ptr<GfxDisplay> gl_make_display(int width,
     lg::error("Could not load icon for OpenGL window");
   }
 
+  glfwSetInputMode(window, GLFW_CURSOR, g_cursor_input_mode);
+  glfwSetInputMode(window, GLFW_STICKY_MOUSE_BUTTONS, GLFW_TRUE);
```

Comment:
Does this mess up imgui menus?

Reply:
It doesn't mess with the imgui. This code snippet will display the cursor when ever the imgui menu is up and disable it when the menu is closed. Although, there is a weird bug where the cursor is there only on startup regardless if imgui is there or not.
https://github.com/animalstyletaco/jak-project/blob/c5e3ab71b595abe5fdf7ca57782844578e3db989/game/graphics/pipelines/opengl.cpp#L100-L102